bip-0085.mediawiki

```diff
@@ -93,7 +97,8 @@ OUTPUT
 
 ==Reference Implementation==
 
-* Python library implementation: [https://github.com/ethankosakovsky/bip85]
+* Python 3.x library implementation: [https://github.com/akarve/bipsea]
```

Comment:
I may be confused here: did you update this implementation to be fully compatible with the current BIP for the xprv format?

Reply:
It will be, as in I will revert the 3.x implementation to use the spec-compliant key/chain code ordering. https://github.com/akarve/bipsea/pull/66

Reply:
Looks like this change should be gated on https://github.com/akarve/bipsea/pull/66?